scitos.ais/scitos.ais.core/src/main/resources/org/hmx/scitos/ais/stylesheet.xsl

```diff
@@ -197,7 +197,7 @@ span.detail-end { float: right; }
                                     </xsl:if>
                                 </td>
                                 <td class="number">
-                                    <xsl:value-of select="count(.//ais:Detail//ais:Token)" />
+                                    <xsl:value-of select="count(.//ais:Detail[@code]/ais:Token)" />
```

Comment:
Overall token count on the results table.

Fix 1: Ensure no double-counting of tokens that are within nested detail annotations. (removing one slash `/`)
Fix 2: Exclude tokens from the total count when they are surrounded by an annotated detail but don't have a specific category assigned to themselves. (checking for `code` attribute's existence)